lua/nvim-treesitter/parsers.lua

```diff
@@ -1480,6 +1480,18 @@ list.smali = {
   maintainers = { "@amaanq" },
 }
 
+list.snakemake = {
+  install_info = {
+    url = "https://github.com/osthomas/tree-sitter-snakemake",
+    branch = "main",
+    location = "tree-sitter-snakemake",
```

Comment:
```suggestion
```

This is only needed if the grammar is in a subdirectory of the repo (which is not the case here).